src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js

```diff
@@ -13,7 +13,7 @@ import FormControl from '@material-ui/core/FormControl';
 import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
 import SchemaPropType from '../../05_pages/NodeForm/SchemaPropType';
 
-import api from './../../../utils/api/api';
+import api from '../../../utils/api/api';
```

Comment:
Is there a reason we need to fix this here?

Reply:
This is just to keep a consistent code like the other `import` statements.